documentation/wiki/Building-Testing-and-Debugging-on-Mono-MSBuild.md

```diff
@@ -1,14 +1,12 @@
 MSBuild can be successfully built on Windows, OS X 10.11, Ubuntu 14.04, and Ubuntu 16.04.
 
-Mono maintains a fork of msbuild (for now) at `https://github.com/mono/msbuild/`. You can clone that and use the `xplat-master` branch or `mono-2017-10` for the next release branch.
+Mono maintains a fork of msbuild (for now) at `https://github.com/mono/msbuild/`. You can clone that and use the `xplat-master` branch or `mono-2018-04` for the next release branch.
 
 # Unix #
 
 **Required packages for OSX & Ubuntu**
 
-MSBuild currently builds with a prerelease version of .NET Core 1.0. It requires the [.NET Core prerequisites](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md), which you can acquire manually or easily get by [installing the .NET Core SDK](https://dot.net/core).
-
-* *OpenSSL*: MSBuild uses the .Net CLI to download Nuget packages during its build process. The CLI requires a recent OpenSSL library available in `/usr/lib`. This can be downloaded using [brew](http://brew.sh/) on OS X (`brew install openssl`) and apt-get (`apt-get install openssl`) on Ubuntu, or [building from source](https://wiki.openssl.org/index.php/Compilation_and_Installation#Mac). If you use a different package manager and see an error that says `Unable to load DLL 'System.Security.Cryptography.Native'`, `dotnet` may be looking in the wrong place for the library.
+MSBuild currently builds with .NET Core 2.1. It requires the [.NET Core prerequisites](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md), which you can acquire manually or easily get by [installing the .NET Core SDK](https://dot.net/core).
```

Comment:
For the mono fork, you just need a recent version of mono installed.

Reply:
So IIUC, there's no need to have the .Net Core Prerequisites or the SDK?

Reply:
I have not tested this on a fresh machine, but yeah. I can build without having the dotnet sdk installed. We use a bootstrap mono+msbuild.

Reply:
@radical how about this '[libunwind](http://www.nongnu.org/libunwind/index.html) is required by .NET Core. Install it using `sudo apt-get install libunwind8`' ?

Reply:
It shouldn't be, but I'm not sure. @directhex would know better.

Reply:
It looks like .NET Core (and its dependencies) are no longer needed.

I'd suggest a note regarding bypassing the Nerdbank.GitVersioning mess (it only works on old distributions without OpenSSL 1.1), though.

Reply:
Nerdbank.GitVersioning should be fixed now with #3657, fortunately.